src/validation.h

```diff
@@ -936,7 +936,6 @@ void ConsolidateRewards(CCustomCSView &view,
                         int height,
                         const std::unordered_set<CScript, CScriptHasher> &owners,
                         bool interruptOnShutdown,
-                        int numWorkers = 0,
                         bool skipStatic = false);
```

Comment:
to prevent further default-value bugs and increase clarity: why not remove the default and have an explicit false on the calls?